Roommatch/Roommatch/ViewControllers/LoginViewController.m

```diff
@@ -0,0 +1,89 @@
+//
+//  LoginViewController.m
+//  Roommatch
+//
+//  Created by Lily Yang on 7/7/22.
+//
+
+#import "LoginViewController.h"
+#import "User.h"
+
+@interface LoginViewController ()
+@property (weak, nonatomic) IBOutlet UITextField *usernameTextField;
+@property (weak, nonatomic) IBOutlet UITextField *passwordTextField;
+
+@end
+
+@implementation LoginViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+}
+
+- (IBAction)signUp:(id)sender {
```

Comment:
If `sender` is not used then you can remove it. Same for below.

Reply:
Had an error when I removed, I think it might be required syntax for button IBAction outlets.